### ISSUE TITLE
o:validateBean on class-level broken in 2.5

### ISSUE THREAD

**Issue by pvydra** [omnifaces/omnifaces#313]
After upgrade to 2.5, o:validateBean stopped working on class-level validation.

```
<h:form>
...  
        <o:validateBean value="#{myBean.agreement}" />

        <p:commandButton update="@form" value="Button" action="# {myBean.save()}"  /> 
</h:form>

```
Stack: Wildfly 9, JSF 2.2, PrimeFaces 5.3

I've found out by debugging that method `ValidateBean.processValidateBean(UIComponent component)` in 2.4 contains following condition:

```
if (!form.equals(getCurrentForm()) || (component instanceof UICommand && !hasInvokedSubmit(component))) {
			return;
}
```

In 2.5 this condition has been changed to this
```
if (!form.equals(getCurrentForm()) || !hasInvokedSubmit(component)) {
			return;
}
```

Since `hasInvokedSubmit`  returns (in both versions) false (because  component is `HtmlForm `but `getCurrentActionSource()` is `UICommand`) the condition always ends execution in 2.5.

**Comment by BalusC** [comment]
That was an awkward mistake. Can you please try today's latest 2.6 snapshot and confirm? Then I will release a 2.5.1.

**Comment by pvydra** [comment]
I can confirm it is working in 2.6-20160920.200801-3

**Comment by BalusC** [comment]
I have just released 2.5.1. It will be on Maven Central sooner or later today.
